packages/cli/src/ceramic-daemon.ts

```diff
@@ -336,6 +342,25 @@ export class CeramicDaemon {
 
     const daemon = new CeramicDaemon(ceramic, opts)
     await daemon.listen()
+
+    // Now that ceramic node is set up we can start publishing metrics
+    if (opts.metrics?.metricsPublisherEnabled) {
+
+      const ipfsVersion = await ipfs.version();
+      ModelMetrics.start({ ceramic: ceramic,
+                       network: params.networkOptions.name,
+                       ceramicVersion: version,
+                       ipfsVersion: ipfsVersion.version,
+                       intervalMS: opts.metrics?.metricsPublishIntervalMS || DEFAULT_PUBLISH_INTERVAL_MS,
+                       nodeId: ipfsId.publicKey, // what makes the best ID for the node?
+                       nodeName: daemon.hostname,
```

Comment:
why have both `nodeName` and `nodeIPAddr` if they are both given the same value?

Reply:
same question for `nodeId` and `nodePeerId`

Reply:
So in the schema, which is intended to be future-extensible, we have the generic concept of nodeId, but right now we don't have a nodeId, so we fill it in with either the PeerID or the IPAddr

Reply:
I kind of think peerId is the right thing to be the unique identifier for a node.  Either that or *maaaaybe* the node DID that we use to authenticate to the CAS.  Do we need another concept of NodeID that is distinct from either of those?

Reply:
@stbrody so the model is here https://github.com/ceramicnetwork/observability/blob/master/packages/model-metrics/src/composites/00-simpleNodeMetrics.graphql

I am trying to make the model itself somewhat future-safe by not making a hard decision on these things - lets sync live to hammer out, am flexible to get this out